pages/common/sed.md

```diff
@@ -18,6 +18,10 @@
 
 `sed '/{{line_pattern}}/s/{{find}}/{{replace}}/' {{filename}}`
 
+- Print only text between n-th line and first upcomming empty line:
```

Comment:
".. between n-th line till the next empty line" sounds better to me

Reply:
Thanks for correction. I have just changed it